### PR TITLE
Start of object browser (#26)

### PR DESCRIPTION
**Pull request by worksofliam**
### Changes

Covers issue #26. This adds an object browser to the IBM i view. I am using the same caching technique used in the member browser by @barrettotte.

Right-click contexts include:

* Run Action

### Checklist

* [X] have tested my change
* [x] updated relevant documentation
* [X] Remove any/all `console.log`s I added
* [X] eslint is not complaining
* [X] have added myself to the contributors' list in the README

### CODE REVIEW

**Comment by worksofliam** [comment]
@barrettotte I have given you 'read' access on a repo that says you can now review PRs. Any chance, whenever you have the time AND if you wanted, you could review this PR for me?

If not, let me know. 

Thanks :)

**Comment by worksofliam** [comment]
Before this is merged, should I add more right-click options?

* Delete
* Change text?

File: src/views/objectBrowser.js [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+    this.resourceUri = vscode.Uri.parse(`${library}/${name}.${type}`).with({scheme: 'object'})
+  }
+}
+
+//https://code.visualstudio.com/api/references/icons-in-labels
+const objectIcons = {
+  'FILE': 'database',
+  'CMD': 'terminal',
+  'MODULE': 'extensions',
+  'PGM': 'chevron-right',
```
Comment (barrettotte) [inline src/views/objectBrowser.js:146]: Is there eventually going to be nested info in objects of type *PGM? If not, maybe setting the icon to something like ```'PGM': 'file-binary'``` would make sense. I really like the other icons though, it feels so natural

Comment (worksofliam) [reply src/views/objectBrowser.js:146]: `file-binary` is much nicer. I will change that.

**Review by barrettotte — COMMENTED** [review]
Code looks awesome and follows the same pattern as the other browsers. I also like the use of the targeted refresh by library. The icons feel very natural minus *PGM, let me know what you think of the comment I left on objectBrowser.js

One thing that I thought of while testing a couple small CL compiles:  Is there any easy way to refresh the object browser after running a compile? That way the newly created PGM would show up in the object list. If not, its not a big deal at all to just refresh

**Comment by worksofliam** [comment]
@barrettotte 

I have fixed an issue where the right-click refresh was not showing up. I also changed the icon to `file-binary`. As well as those, if you have auto-refresh enabled, it will invoke`refreshObjectList` after an action has finished executing.

Please review again :)

**Review by barrettotte — APPROVED** [review]
Just tried compiling again and the appropriate library refreshed on compile, thanks!

**Comment by worksofliam** [comment]
Will be merging today! Thanks @barrettotte :)

**Pull request merged**